src/MoBi.Presentation/Tasks/Interaction/MoleculeStartValuesTask.cs

```diff
@@ -78,10 +78,11 @@ private MoleculeStartValuesBuildingBlock createStartValuesBasedOnUsedTemplates(M
 
       public override MoleculeStartValuesBuildingBlock CreateStartValuesForSimulation(SimulationConfiguration simulationConfiguration)
       {
-         var simulationStartValues = _startValuesCreator.CreateFrom(simulationConfiguration.SpatialStructure, simulationConfiguration.Molecules)
-            .WithName(simulationConfiguration.MoleculeStartValues.Name);
-
-         var templateValues = UpdateValuesFromTemplate(simulationStartValues, simulationConfiguration.MoleculeStartValues);
+         //TODO OSMOSES combining multiple spatial structures and molecule building blocks is not supported yet
+         var simulationStartValues = _startValuesCreator.CreateFrom(simulationConfiguration.SpatialStructures.First(), simulationConfiguration.Molecules.First())
```

Comment:
For now this just uses the first SS and Molecules. This probably will be implemented when we are combining module building blocks